itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -1539,15 +1539,14 @@ public void testCheckPointingWithSourceTableDataInserted() throws Throwable {
             .run("insert into t2 values (24)")
             .run("insert into t1 values (4)")
             .dump(primaryDbName, dumpClause);
-
+    assertEquals(modifiedTimeTable1CopyFile, fs.listStatus(tablet1Path)[0].getModificationTime());
```

Comment:
Why is this changed?

Reply:
because data is moved now. We are checking this late